tests/python_package_test/test_dask.py

```diff
@@ -1527,11 +1527,11 @@ def test_predict_with_raw_score(task, output, cluster):
             assert_eq(raw_predictions, pred_proba_raw)
 
 
-def test_distributed_quantized_training(cluster):
+def test_distributed_quantized_training(tmp_path, cluster):
     with Client(cluster) as client:
         X, y, w, _, dX, dy, dw, _ = _create_data(objective="regression", output="array")
 
-        np.savetxt("data_dask.csv", np.hstack([np.array([y]).T, X]), fmt="%f,%f,%f,%f,%f")
+        np.savetxt(tmp_path / "data_dask.csv", np.hstack([y[:, None], X]), fmt="%f,%f,%f,%f,%f")
```

Comment:
I'm confused.... why did routing this file to a different path also require any changes to this `np.hstack()` call. Could you please explain this?

Reply:
This is a totally different change, we don't need it. I just spotted that the way of creating a column axis for `y` seemed quite confusing. It's one the same line so I crept the change in. Can be reverted.

Reply:
thanks, please revert it.